src/Castle.Windsor/MicroKernel/SubSystems/Configuration/DefaultConfigurationStore.cs

```diff
@@ -181,7 +181,8 @@ public IConfiguration[] GetInstallers()
 
 		public IResource GetResource(String resourceUri, IResource resource)
 		{
-			if (resourceUri.IndexOf(Uri.SchemeDelimiter) == -1)
+			var uriSchemeDelimiter = "://";
```

Comment:
Cool cool... just I have been here before and was wondering if we wanted create a shim for it. If not all good...